output/output.go

```
@@ -0,0 +1,40 @@
+package output
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/viper"
+)
+
+func Println(data interface{}) (int, error) {
+	if viper.GetBool("cli.quiet") {
+		return 0, nil
+	}
+
+	return fmt.Println(data)
+}
+
+func Printf(format string, objects ...interface{}) (int, error) {
+	if viper.GetBool("cli.quiet") {
+		return 0, nil
+	}
+
+	return fmt.Printf(format, objects...)
+}
+
+func ErrorPrintln(data interface{}) (int, error) {
+	if viper.GetBool("cli.quiet") {
```

I wonder if it's wise to swallow errors, too, when using `--do-not-make-a-noise`?